retriever/lib/create_scripts.py

```diff
@@ -1,11 +1,11 @@
 """Module to create scripts"""
 import collections
-import os
 import json
+import os
```

Comment:
I think the imports should remain as what is in the master branch and import os is not being used